packages/amplify-category-api/src/provider-utils/awscloudformation/utils/database-url.ts

```diff
@@ -0,0 +1,28 @@
+import { URL } from 'url';
+import { ImportedRDSType, RDSDataSourceConfig } from '@aws-amplify/graphql-transformer-core';
+
+export const parseDatabaseUrl = (databaseUrl: string): Partial<RDSDataSourceConfig> => {
+  try {
+    const parsedDatabaseUrl = new URL(databaseUrl);
+    const [username, password] = [parsedDatabaseUrl.username, parsedDatabaseUrl.password];
```

Comment:
can't these two lines just be `const { username, password } = (new URL(databaseUrl));` ? The list destructuring reads a little bizarrely, and isn't really necessary.

Reply:
That's right. Refactored this to read better.